src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/PartialSuccessTransmissionPolicy.cs

```diff
@@ -0,0 +1,153 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation
+{
+    using System;
+    using System.Threading.Tasks;
+
+    using System.Web.Script.Serialization;
+
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation;
+
+#if NET45
+    using TaskEx = System.Threading.Tasks.Task;
+#endif
+
+    internal class PartialSuccessTransmissionPolicy : TransmissionPolicy, IDisposable
+    {
+        private readonly JavaScriptSerializer serializer = new JavaScriptSerializer();
+
+        private TaskTimer pauseTimer = new TaskTimer { Delay = TimeSpan.FromSeconds(SlotDelayInSeconds) };
```

Comment:
In half. Base class does not need it and I do not want to create and dispose it if I do not need it; it is an expensive resource